enhancements/sig-architecture/26-policy-generator/README.md

```diff
@@ -0,0 +1,266 @@
+ # The categories to be used in the
+ # policy.open-cluster-management.io/categories annotation. This
+ # defaults to "CM Configuration Management".
+ # categories: []
+ # The controls to be used in the
+ # policy.open-cluster-management.io/controls annotation. This
+ # defaults to
+ # "CM-2 Baseline Configuration".
+ controls:
+   - PR.DS-1 Data-at-rest
+ # The namespace of all the policies. This is required.
+ namespace: my-policies
+ # The placement (i.e. PlacementRule) configuration for the
+ # policies. This defaults to a placement configuration that
+ # matches all clusters.
+ placement:
+   # To generate a placement rule per policy, you may set the
+   # cluster selectors as shown below.
+   # clusterSelectors:
+   #   cloud: red hat
+   # A name may also be specified to consolidate placement rules with
+   # the same cluster selectors.
+   # name: my-placement-rule
+   # To reuse an existing placement rule, you may specify the path
+   # here. This placement rule will be used by all policies by default.
+   placementRulePath: examples/native/input/plr.yaml
+ # The remediation action (inform or enforce) for the policy. This
+ # defaults to "inform".
+ remediationAction: enforce
+ # The severity of the policy violation. This defaults to "low".
+ severity: medium
+ # The standards to be used in the
+ # policy.open-cluster-management.io/standards annotation. This
+ # defaults to "CM-2 Baseline Configuration".
+ # standards: []
+policies:
+ # The name of the first policy to create. This is required.
+- name: policy-app-config
+  # Determines if the policy is disabled. This defaults to false.
+  disabled: false
+  # The list of Kubernetes resource object manifests to include in
+  # the policy. This is required.
+  manifests:
+    # This can specify a single file or a flat directory of files.
+    - path: input/configmap.yaml
+ # Determines if all the manifests are in a single
+ # ConfigurationPolicy in the policy. This defaults to false.
+ # singleConfigurationPolicy: false
+ remediationAction: enforce
+- name: policy-app-config2
+  disabled: true
+  manifests:
+    - path: some-directory
+```
+
+#### Example Workflow
+
+- A security engineer, SRE, infrastructure engineer, or etc. creates a pull request (PR) with the
+  Kubernetes resource object manifests that should be wrapped in a policy.
+- Based on the organization policy, the PR is reviewed and merged in the organization's Git
+  repository.
+- A security engineer, SRE, infrastructure engineer, or etc. creates a PR with the
+  `kustomization.yaml` file, the generator plugin configuration, and any Kustomize patches they
+  would like to apply to the generated policies.
+- Based on the organization policy, the PR is reviewed and merged in the organization's Git
+  repository.
+- A SRE then configures an ACM subscription to apply the generated policies using GitOps. This
+  subscription would be limited to just the policy directory created in the previous step.
+
+## Design Details
```

Comment:
All of this seems like boiler plate. Should we remove it or replace it?

Reply:
It is--I'll take it out since I wasn't sure what should have gone in there for us.